task02-pixel-perfect/task02.css

```diff
@@ -0,0 +1,57 @@
+/**
+* All Rights Reserved
+* This software is property information of AKUREY S.A.
+* Contact us at contact@akurey.com
+* @summary task 02 style sheet
+*/
+.container {
+    display: grid;
+    grid-template-columns: repeat(4,auto);
+    column-gap: 20px;
+    margin: auto;
+    width: 940px;
+    height: 279.65px;
```

Comment:
The decimals are not necessary here, only if it's strictly needed, in this case, you can notice using 280px will have the same result with clearer more readable code.